spec/models/miq_worker/container_common_spec.rb

```diff
@@ -1,11 +1,18 @@
 describe MiqWorker::ContainerCommon do
+  before { EvmSpecHelper.local_miq_server }
+  let(:compressed_server_id) { MiqServer.my_server.compressed_id }
+
+  def deployment_name_for(name)
+    "#{compressed_server_id}-#{name}"
+  end
```

Comment:
Should this just be part of the ContainerCommon as a class method?  Then the specs can share it?